data/projects/implement_aspects.md

```diff
@@ -7,9 +15,26 @@ The aspects concept exists as
 already possible to define aspects and use them in bears - however this has no
 effect because coala doesn't understand what to do with them yet.
```

Comment:
Maybe give a short rundown of what aspects are and how they work. as in
"A Bear can suport an aspect, eg code clone detection, which belongs to the category xy. If a user wants code clone detection for their lang, instead of using somebearthatdoesclonedetectionfortheirlang they can simply add the clonedetection aspect and coala figures out if there is a bear supporting it for the lang and uses it.